testapps/test_pillowtop/tests/test_reindexer.py

```diff
@@ -0,0 +1,74 @@
+import uuid
+from django.core.management import call_command
+from django.test import TestCase
+from casexml.apps.case.models import CommCareCase
+from casexml.apps.case.signals import case_post_save
+from corehq.apps.es import UserES, CaseES, FormES, ESQuery
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.tests.utils import TestFormMetadata
+from corehq.pillows.mappings.user_mapping import USER_INDEX
+from corehq.util.context_managers import drop_connected_signals
+from corehq.util.elastic import delete_es_index
+from corehq.util.test_utils import make_es_ready_form
+
+
+class PillowtopReindexerTest(TestCase):
+
+    domain = 'reindex-test-domain'
+
+    def test_user_reindexer(self):
+        username = 'reindex-test-username'
```

Comment:
I don't have to because the reindexer deletes it when it starts (and I delete it in the other test that needs it). probably would be cleaner/safer to just delete it anyways though.